qa/integration/services/dockerized/Dockerfile

```diff
@@ -16,6 +16,11 @@ USER tester
 ENV WORKDIR /home/tester
 WORKDIR $WORKDIR
 
+# Generate TLS certificates that can be used by services.
+ENV CERTIFICATES_DIR $WORKDIR/certificates
+RUN mkdir -p $CERTIFICATES_DIR && \
```

Comment:
@wainersm I thought about this a bit this morning ... I think we may be introducing a strange bug here, given that this is our base image.
Couldn't we potentially run into a situation (not on CI but locally) where we build two child images from different versions of the base?

Honestly ... why not simply commit one key for tests to the repo and not have any noise from this stuff? We can then even access it from host and docker images trivially by just `ADD`ing it here instead of generating it?